### ISSUE TITLE
docker compose example for self-hosting

### ISSUE THREAD

**Issue by azumukupoe** [screeny05/letterboxd-list-radarr#14]
I think it would help many who are having trouble with heroku including me

**Comment by screeny05** [comment]
Has been added to the readme. Please let me know if this works for you.

If there are problems with the setup, please feel free to create a new issue.

**Comment by azumukupoe** [comment]
This is what I'm getting:

```
npm ERR! code 127

npm ERR! path /home/node/app

npm ERR! command failed

npm ERR! command sh -c nodemon --watch lib --watch index.ts -e js,ts --exec 'npm run dev'
```

**Comment by screeny05** [comment]
Oh, i just noticed, that i forgot to add an instruction to the readme. Could you please follow these steps and give me feedback if it works?

```
git clone 
cd letterboxd-list-radarr
npm install
docker-compose up -d
```

**Comment by azumukupoe** [comment]
> npm install

Why didn't I try this...
Well thanks and you probably  should add that to readme.

**Comment by screeny05** [comment]
Done. Did this work for you?

**Comment by azumukupoe** [comment]
Yes, though sometimes Node crashes when trying to fetch bigger lists for the first time.

**Comment by screeny05** [comment]
Is there an error-message with which it crashes?

Can you provide me a link to a list like this for testing purposes?

**Comment by azumukupoe** [comment]
```
Error: aborted

at connResetException (node:internal/errors:631:14)

at TLSSocket.socketCloseListener (node:_http_client:442:27)

at TLSSocket.emit (node:events:388:22)

at node:net:661:12

at TCP.done (node:_tls_wrap:564:7)

npm ERR! code 1

npm ERR! path /home/node/app

npm ERR! command failed

npm ERR! command sh -c node --inspect=5858 -r ts-node/register index.ts


npm ERR! A complete log of this run can be found in:

npm ERR! /home/node/.npm/_logs/2020-12-01T23_23_05_789Z-debug.log

[nodemon] app crashed - waiting for file changes before starting..
```
Have to restart compose after each crash.

Can't say which one; it's pretty random.
It once crashed with a relatively small list.

Try:
https://letterboxd.com/year-in-review/
https://letterboxd.com/jbutts15/list/the-complete-criterion-collection/
https://letterboxd.com/kjknow/list/new-beverly-cinema-films-by-quentin-tarantino/
